cpp/tests/utilities/column_wrapper.hpp

```diff
@@ -199,9 +295,11 @@ auto make_chars_and_offsets(StringsIterator begin, StringsIterator end, Validity
  * @brief `column_wrapper` derived class for wrapping columns of fixed-width
  * elements.
  *
- * @tparam Element The fixed-width element type
+ * @tparam ElementTo The fixed-width element type that is created
+ * @tparam SourceElementT The fixed-width element type that is used to create elements of type
+ * `ElementTo`
  **/
-template <typename ElementTo>
+template <typename ElementTo, typename SourceElementT = ElementTo>
```

Comment:
Use consistent type naming
```suggestion
template <typename ElementTo, typename ElementFrom = ElementTo>
```

Reply:
`ElementFrom` is used in one of the template constructor, and using this will shadow that, and thus a different name was chosen.

https://github.com/rapidsai/cudf/blob/branch-0.15/cpp/tests/utilities/column_wrapper.hpp#L293